crates/sui-types/src/messages.rs

```diff
@@ -1992,6 +1992,8 @@ pub struct TransactionEffects {
     /// Object Refs of objects now deleted (the old refs).
     pub deleted: Vec<ObjectRef>,
     /// Object refs of objects now wrapped in other objects.
+    pub unwrapped_then_deleted: Vec<ObjectRef>,
```

Comment:
You also need to update https://github.com/MystenLabs/sui/blob/main/crates/sui-json-rpc-types/src/lib.rs#L1905

Reply:
This change still needs to be done right?